client/app/components/empty-state/EmptyState.jsx

```diff
@@ -17,78 +17,45 @@ function createDashboard() {
   });
 }
 
-function getSteps({
-  showAlertStep,
-  showDashboardStep,
-  showInviteStep,
-}) {
+function Step({ completed, text, url = null, onClick = null, urlText, show = true }) {
```

Comment:
One question: Does it make a difference setting `url` and `onClick` to `null`, isn't that a default?

I liked it with `Step` component, though I myself am tendentious to use ES6 syntax haha:
```javascript
const Step = ({ completed, text, url, onClick, urlText, show = true }) => (
  show &&
  <li className={classNames({ done: completed })}>
    {onClick && (
      <a href="#" onClick={onClick}>
        {urlText}
      </a>
    )}
    {url && <a href={url}>{urlText}</a>}
    {text}
  </li>
);
```

Reply:
> Does it make a difference setting url and onClick to null

By default they will be `undefined`. In this case there's no difference, because `undefined == false` and `null == false`.

Reply:
This is my pref:

```jsx
function Step({ show, completed, text, url, urlText, onClick }) {
  if (!show) {
    return null;
  }

  return (
    <li className={classNames({ done: completed })}>
      <a href={url || 'javascript:void(0)'} onClick={onClick}>{urlText}</a> {text}
    </li>
  );
}
```
No need for conditional spacing, empty onClick simply doesn't register, clear conditional exit at top of method.

Reply:
@gabrieldutra you're correct. The `null` default is pointeless. That's what I get for writing code at the middle of the night... :)

@ranbena 👍